plugins/helm/parser/include/parser/yamlparser.h

```diff
@@ -0,0 +1,105 @@
+#ifndef CC_PARSER_YAML_PARSER_H
+#define CC_PARSER_YAML_PARSER_H
+
+#include <atomic>
+
+#include <parser/abstractparser.h>
+#include <parser/parsercontext.h>
+
+#include <util/parserutil.h>
+
+#include "yaml-cpp/yaml.h"
+
+namespace cc
+{
+namespace parser
+{
+class YamlParser : public AbstractParser
```

Comment:
While the plugin directory was renamed to `helm`, the parser and service classes kept the name `YamlParser` and `YamlService`. This is highly confusing.